src/app/index.tsx

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 import '../scss/application.scss';
 
 import SignUp from './screens/SignUp';
+import Login from './screens/Login';
 
 function App() {
-  return <SignUp />;
+  return (
+    <Router>
+      <Switch>
+        <Route path="/sign_up">
```

Comment:
consider to add this paths on a constant file, could be on `~constants/paths.ts` or something

```
export const PATHS = {
  login: '/login'
  signUp: '/sign_up'
}

```